lisp/ess-mode.el

```diff
@@ -100,9 +100,9 @@
     (define-key map "\C-c\C-k"   'ess-force-buffer-current)
     (define-key map "\C-c`"      'ess-show-traceback)
     (define-key map [(control ?c) ?~] 'ess-show-call-stack)
-    (define-key map "{"          'ess-electric-brace)
-    (define-key map "}"          'ess-electric-brace)
     (define-key map "\C-\M-q"    'ess-indent-exp)
+    (define-key map "{"          #'skeleton-pair-insert-maybe)
```

Comment:
Any special reason to quote them as functions (#')?

Reply:
Habit, more than anything else. I don't think it'll make a difference (except this way the byte compiler can warn if they aren't defined). I know `"'` looks weird to some people though. Do you want me to remove the sharp quote?

Reply:
I do use #' when it does matter, for keymaps I think everyone use simple quite. My concern is that it should be either #' or ' everywhere in keymaps. Otherwise it's ugly and people raise questions. I think It won't work for nested keymaps, those are not functions. So let's stick to ' throughout. Did you introduced a couple of these in the earlier PRs?

Reply:
OK, switched it back to `'`. I don't *think* I did this elsewhere?